SEED-platform/seed
# 2594 feat: docs page

#### Any background context you want to provide?
SEED is preparing for a public release. Part of that effort is improving documentation.

#### What's this PR do?
Adds a public documentation page, which displays a list of frequently asked questions. Also, it updates the sidebar so that the hamburger menu is there instead of in the top navbar (see screenshots).

The current FAQ items were pulled directly from https://seed-platform.org/

#### How should this be manually tested?
Before testing, rerun pip install, this PR adds a python dependency.

Go to the documentation page (logged in and not) at `/documentation` or by clicking the "book" icon in the left sidebar. If not logged in, all of the navbar icons should be appropriately disabled.
You should see a list of faq categories, each expandable where you can then expand and view questions and their response by clicking on the question.
You should be able to use the search bar to filter by any word in the question/response, as well as by any hidden "tag" (tags are specified in the markdown frontmatter, in the docs/faq/<category>/<faq item>.md file.

You can also try creating new markdown files and see how they're added to the list of FAQ items.

#### What are the relevant tickets?
#2594 

#### Screenshots (if appropriate)
<img width="939" alt="Screen Shot 2021-02-26 at 1 21 16 PM" src="https://user-images.githubusercontent.com/18518728/109351965-57698680-7837-11eb-920a-7a7ef8da7081.png">
<img width="939" alt="Screen Shot 2021-02-26 at 1 21 36 PM" src="https://user-images.githubusercontent.com/18518728/109351972-59cbe080-7837-11eb-8a8f-4751f37755ec.png">
<img width="938" alt="Screen Shot 2021-02-26 at 1 21 54 PM" src="https://user-images.githubusercontent.com/18518728/109351977-5c2e3a80-7837-11eb-9012-47fc7eb0ceb3.png">
